### PR TITLE
Update security.md

### PR DESCRIPTION
**Pull request by Satyam3976**
Fixes issue: #1064 
I have updated the content of heading ###objectives and ###Specific thigs to learn
Also, I moved the previous content under a new heading ###Skills developed

### CODE REVIEW

**Comment by Satyam3976** [comment]
@Kundan28 Thank you sir

**Comment by Satyam3976** [comment]
@Kundan28 If the PR is fine, then please merge it.It is approved by every mentor now

**Comment by Kundan28** [comment]
@Satyam3976 Only project admins could merge the PR. So I would request you to wait until you get any response fro their side.

**Comment by alodahl** [comment]
@Satyam3976 will you be able to fix this issue before the end of GSSoC in 2 weeks?

**Comment by Satyam3976** [comment]
Yes, I will do by tomorrow

**Comment by vegetabill** [comment]
@Satyam3976 just a reminder I'll be closing this PR Sunday, 24 May unless revisions come in.

File: career/roles-in-tech/security.md [hunk 1]
```diff
@@ -35,22 +35,22 @@ Apprentices will learn secure development basics, common pitfalls, and how to av
 
 _Participants will be able to learn:_
```
Comment (alodahl) [inline career/roles-in-tech/security.md:36]: These the Objectives should list what someone will know once they finish reading this outline.  Do you agree that this is what's listed?

File: career/roles-in-tech/security.md [hunk 2]
```diff
@@ -35,22 +35,22 @@ Apprentices will learn secure development basics, common pitfalls, and how to av
...
-- How to implement secure n/w architecture and system design
-- Discover security vulnerabilities across an entire network
+- provide system, network and security knowledge
+- analyze and define security requirements
+- protect integrity, confidentiality and availability an information system
+- implement secure n/w architecture and system design
+- discover security vulnerabilities across an entire network
+- develop skills like - risk management, Identity and Access Management, Programming Skills, Communication Skills
 
 ### Specific Things to Learn
```
Comment (alodahl) [inline career/roles-in-tech/security.md:45]: This section is for things participants will learn by reading this outline.

File: career/roles-in-tech/security.md [hunk 3]
```diff
@@ -35,22 +35,22 @@ Apprentices will learn secure development basics, common pitfalls, and how to av
...
 
 _The basic required skills as a Security Engineer are_
```
Comment (alodahl) [inline career/roles-in-tech/security.md:47]: This should be a different section under Lesson.  But "The basic required skills as a Security Engineer" could be one of the Specific Things to Learn.  This should just be a list/summary of what's ahead - no one should actually learn anything in this section.

**Comment by Kundan28** [comment]
@Satyam3976  Are you still working on this ? Please complete it before 31st May so that we have sufficient time to review it.

**Comment by alodahl** [comment]
Please feel free to ask questions if you still don't know what this task requires.

File: career/roles-in-tech/security.md [hunk 4]
```diff
@@ -61,6 +59,10 @@ After gaining basic skills one can go further as _Security Architect_ or _Securi
 - [Techtonica's Roles in Tech Presentation](https://drive.google.com/file/d/1oXgOX1EC4igI6ZwgJlhF6AN4dG2Yn_S4/view) (security section starts at 2:57)
 - [Slideshow - Roles in Tech : Security](https://docs.google.com/presentation/d/1DjbfFt-rgQzBBPwVSXL1nGww1ziycJIxiSRmWbS-v40/edit#slide=id.p)
 
+### Lesson
+
+_The basic required skills as a Security Engineer are_
```
Comment (alodahl) [inline career/roles-in-tech/security.md:64]: If you were reading this outline, would this line make sense?

**Comment by alodahl** [comment]
Sorry, we discussed the issue requirements quite a bit, and you've had this issue for a month.  I am closing this PR.

**Comment by Kundan28** [comment]
Removed from scoresheet too.

**Pull request closed without merging**